src/cloud-cli/userdb.ts

```diff
@@ -156,18 +156,18 @@ export function migrate(): number {
 
   if (create_db) {
     logger.info(`Creating database ${userdbname}`);
-    const cmd = `createdb -h ${configFile.database.hostname} -p ${configFile.database.port} ${userdbname} -U ${configFile.database.username} -w ${configFile.database.password} -e`;
+    const cmd = `PGPASSWORD=${configFile.database.password} createdb -h ${configFile.database.hostname} -p ${configFile.database.port} ${userdbname} -U ${configFile.database.username} -ew ${userdbname}`;
```

Comment:
Bug in `createdb`: the positional parameter should be the DB name. And `PGPASSWORD` needs to be available in the command's environment